### ISSUE TITLE
[Lyrics Plus] Lyrics not change when locking modes and changing songs

### ISSUE THREAD

**Issue by surya27895** [spicetify/spicetify-cli#2318]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.2.10.760.g52970952
- Spicetify version: v2.18.0
```

### 📝 Description

Like lyrics not fetching again and not changing even song has change
not have this problem at v2.17.2
affected all mode, karaoke, synced , unsynced, and genius
need change to another tab other than lyrics plus tab and back to lyrics plus tab again to make it fetch new lyrics, but sometimes still no luck

already do fresh install spotify and spicetify but still same

### 📸 Screenshots

![image](https://user-images.githubusercontent.com/22859567/235969111-08a509f0-ffe1-4e4c-9f98-d932a3fbdf00.png)
![image](https://user-images.githubusercontent.com/22859567/235969149-52b575e5-4be9-4f26-9e07-e29cd0014a9c.png)

**Comment by celishere** [comment]
I have the same

**Comment by rxri** [comment]
I can't reproduce, could you do `spicetify restore backup apply` and try again?

**Comment by ohitstom** [comment]
this was fixed as part of #2293

**Comment by Lseoksee** [comment]
This problem does not change when you play the next song when you look at the lyrics with the way the lyrics are fixed.

**Comment by surya27895** [comment]
After some PR above, i try replace with the file manualy  but still no luck for me even at first song after apply-ing spicetify

**Comment by kyrie25** [comment]
> After some PR above, i try replace with the file manualy but still no luck for me even at first song after apply-ing spicetify

Fix is not yet merged.

**Comment by surya27895** [comment]
> > After some PR above, i try replace with the file manualy but still no luck for me even at first song after apply-ing spicetify
> 
> Fix is not yet merged.

I download it from your PR, the index.js file to test it, but still same

**Comment by kyrie25** [comment]
> > > After some PR above, i try replace with the file manualy but still no luck for me even at first song after apply-ing spicetify
> > 
> > 
> > Fix is not yet merged.
> 
> I download it manualy from your PR, the index.js file and replace mine to test it, but still same

Can you list the steps you made to reproduce the issue? All the people on our team were only able to replicate your issue by playing previous tracks of which the PR above resolved it.

**Comment by kyrie25** [comment]
Clone the latest file from the commit also.

**Comment by surya27895** [comment]
> > > > After some PR above, i try replace with the file manualy but still no luck for me even at first song after apply-ing spicetify
> > > 
> > > 
> > > Fix is not yet merged.
> > 
> > 
> > I download it manualy from your PR, the index.js file and replace mine to test it, but still same
> 
> Can you list the steps you made to reproduce the issue? All the people on our team were only able to replicate your issue by playing previous tracks of which the PR above resolved it.
> 
> If possible, run `spicetify enable-devtools`, press `Ctrl + Shift + I` while focus on the Spotify client and screenshot the `Console` tab while reproducing the issue also.

it seems only occur when locking mode activated on lyrics plus, when use non locking mode seems it run normally, as seen at this video it might error occur when fail to fetch lyrics on karaoke mode then affected another mode
https://user-images.githubusercontent.com/22859567/236119333-f5dc1159-1bd2-4f06-9ee5-f6912ed6c5ca.mp4

and here screenshoot from console
![image](https://user-images.githubusercontent.com/22859567/236119104-b0437df2-6a02-417a-8b46-99ef03be17ec.png)
![image](https://user-images.githubusercontent.com/22859567/236119177-1448b18e-276d-4496-9b97-49a936e7e849.png)

**Comment by kyrie25** [comment]
Can you try the latest commit again? It seems to have fixed it on my end.

**Comment by surya27895** [comment]
> Can you try the latest commit again? It seems to have fixed it on my end.

can confirm last commit fit it. Thanks!!!

**Comment by kyrie25** [comment]
> > Can you try the latest commit again? It seems to have fixed it on my end.
> 
> 
> 
> can confirm last commit fix it. Thanks!!!
> 
> but it seems make the karaoke mode not fetching the karaoke lyrics for me after changing song if fail get karaoke lyrics, but not broke the another mode 😅

I'm having trouble understanding what you're trying to say. Can you rephrase that?

**Comment by surya27895** [comment]
> > > Can you try the latest commit again? It seems to have fixed it on my end.
> > 
> > 
> > can confirm last commit fix it. Thanks!!!
> > but it seems make the karaoke mode not fetching the karaoke lyrics for me after changing song if fail get karaoke lyrics, but not broke the another mode 😅
> 
> I'm having trouble understanding what you're trying to say. Can you rephrase that?


https://user-images.githubusercontent.com/22859567/236126199-cdbcd409-6620-4895-97b8-db40ab9a5285.mp4

I hope this video can explain better than my word
but the problem is again, there is a problem that affected karaoke lyrics fetching fail when it not found and that condition will affecting process to fetching new karaoke lyrics that never caching before

**Comment by kyrie25** [comment]
Can you screenshot the console logs also?

**Comment by kyrie25** [comment]
Can you try the latest commits again and screenshot the console logs? I'm currently not available atm so I'll keep in mind for any additional information you can provide.

**Comment by surya27895** [comment]
here the screenshot use `index.js` from commit [29e38a0](https://github.com/spicetify/spicetify-cli/pull/2321/commits/29e38a01eef1c8579f8ab3b3edfc637d8458b33e)
![image](https://user-images.githubusercontent.com/22859567/236135055-3a82677f-972b-474b-a6e5-861f409ea468.png)
![image](https://user-images.githubusercontent.com/22859567/236135141-b467b30f-6056-40d2-b36a-6dbfeb3271d7.png)
![image](https://user-images.githubusercontent.com/22859567/236135209-304723d5-c658-4243-9bae-ea7420a03e16.png)

**Comment by kyrie25** [comment]
> here the screenshot from the `console` using `index.js` from commit [29e38a0](https://github.com/spicetify/spicetify-cli/pull/2321/commits/29e38a01eef1c8579f8ab3b3edfc637d8458b33e)

So that was caused because I made a commit from my phone lol. Should be fixed now.

**Comment by XanoPhobious** [comment]
This bug still happens. But, it seems only when I pin the Synced tab AND only when the lyrics is provided by Netease.

**Comment by surya27895** [comment]
> This bug still happens. But, it seems only when I pin the Synced tab AND only when the lyrics is provided by Netease.

this bug need manual works to fix at this moment since it not merged yet
if you want try you can download `index.js` file from commit https://github.com/spicetify/spicetify-cli/pull/2321/commits/c5dc16a4a67f21c99b1fe77c5d847697926a5ad1 and replace yours
